### PR TITLE
Make level-8.

### PR DESCRIPTION
**Pull request by creator-max**
Add a Repair service. Add orders. Use Dictionary<T, V>.

### CODE REVIEW

File: AutoPark/CarRepair/Orders.cs [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+            _details = new List<string>();
+            OrdersCount = new Dictionary<string, int>();
+            LoadOrders(ordersPath);
+        }
+
+        public Dictionary<string, int> OrdersCount { get; private set; }
+
+        public void Print()
+        {
+            foreach(var key in OrdersCount.Keys)
```
Comment (kuzmiich) [inline AutoPark/CarRepair/Orders.cs:23]:
```suggestion
            foreach(var (key, value) in OrdersCount)
```
Use deconstructor. It`s better

**Review by kuzmiich — APPROVED** [review]
Well done

File: AutoPark/CarRepair/Orders.cs [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        private List<string> _details;
+
+        public Orders(string ordersPath)
+        {
+            _details = new List<string>();
+            OrdersCount = new Dictionary<string, int>();
+            LoadOrders(ordersPath);
+        }
+
+        public Dictionary<string, int> OrdersCount { get; private set; }
```
Comment (PushMeNow) [inline AutoPark/CarRepair/Orders.cs:19]: Name it Orders.

File: AutoPark/CarRepair/Orders.cs [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+        public void Print()
+        {
+            foreach(var (key, value) in OrdersCount)
+            {
+                Console.WriteLine($"{key, -7}:{value, 3}шт.");
+            }
+        }
+
+        public void AddOrder(string detail)
```
Comment (PushMeNow) [inline AutoPark/CarRepair/Orders.cs:29]: private

**Review by PushMeNow — CHANGES_REQUESTED** [review]

File: AutoPark/CarRepair/Orders.cs [hunk 4]
```diff
@@ -0,0 +1,67 @@
+﻿using AutoPark.CsvHelper;
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace AutoPark.CarRepair
+{
+    public class Orders
```
[the remark this answers is not in the document]
Comment (creator-max) [reply AutoPark/CarRepair/Orders.cs]: Fixed

File: AutoPark/CarRepair/Orders.cs (hunk 2 above)
Comment (creator-max) [reply AutoPark/CarRepair/Orders.cs]: Fixed

**Review by PushMeNow — APPROVED** [review]

**Pull request merged**